### PR TITLE
Tip 1 edits

### PR DESCRIPTION
**Pull request by rasbt**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**

I made various edits but tried the keep everything in there and leave the paragraph structure intact. So, they are mostly minor changes.

### CODE REVIEW

**Review by Benjamin-Lee — APPROVED** [review]
Excellent edits, as always. LGTM

File: content/03.when-to-use-dl.md [hunk 1]
```diff
@@ -1,47 +1,48 @@
...
-Similarly, Chen et al. [@doi:10.1038/s41746-019-0122-0] tested deep learning and a variety of traditional machine learning methods such as logistic regression and random forests on five different clinical datasets, and found that traditional methods matched or exceeded the accuracy of the deep learning model in all cases despite requiring an order of magnitude less training time.
+The researchers found that while well-tuned deep learning approaches generally tend to outperform conventional classifiers, simple methods such as Naive Bayes classification tend to outperform deep learning as the dataset's noise increases.
+Similarly, Chen et al. [@doi:10.1038/s41746-019-0122-0] tested deep learning and a variety of traditional machine learning methods such as logistic regression and random forests on five different clinical datasets. They found that traditional methods matched or exceeded the accuracy of the deep learning model in all cases despite requiring an order of magnitude less training time.
```
Comment (Benjamin-Lee) [inline content/03.when-to-use-dl.md:44]:
```suggestion
Similarly, Chen et al. [@doi:10.1038/s41746-019-0122-0] tested deep learning and a variety of traditional machine learning methods such as logistic regression and random forests on five different clinical datasets.
They found that traditional methods matched or exceeded the accuracy of the deep learning model in all cases despite requiring an order of magnitude less training time.
```

File: content/03.when-to-use-dl.md [hunk 2]
```diff
@@ -1,47 +1,48 @@
...
-For example, automated machine learning (AutoML) tools, such as TPOT [@doi:10.1007/978-3-319-31204-0_9] and Turi Create [@https://github.com/apple/turicreate], are able to automatically test and optimize multiple machine learning models, and can allow users to achieve competitive performance with only a few lines of code.
-Thankfully, there are efforts underway to extend these and other automation frameworks in order to reduce the expertise required to build and use deep learning models as well.
-For example, TPOT, Turi Create and AutoKeras [@arxiv:1806.10282], are already capable of abstracting away much of the programming required for "standard" deep learning tasks, and high level interfaces such as Keras [@https://keras.io], make it increasingly straightforward to design and test custom deep learning architectures
+Conversely, traditional machine learning methods are generally more straightforward to implement. There are currently more tools for automating the model selection and training process for traditional machine learning models than for deep learning models.
```
Comment (Benjamin-Lee) [inline content/03.when-to-use-dl.md:25]:
```suggestion
Conversely, traditional machine learning methods are generally more straightforward to implement.
There are currently more tools for automating the model selection and training process for traditional machine learning models than for deep learning models.
```

File: content/03.when-to-use-dl.md [hunk 3]
```diff
@@ -1,47 +1,48 @@
...
-On the whole, initial evaluation as to whether similar problems (including analogous ones in other domains) have been solved successfully using deep learning can inform researchers as to the potential for deep learning to address their needs.
+For example, Ferreira et al. used deep learning to recognize individual birds from images [@doi:10.1111/2041-210X.13436] despite this problem being very difficult historically.
+By combining automatic data collection using RFID tags with data augmentation and transfer learning (explained in [Tip 5](#architecture)), the authors were able to use deep learning to achieve 90% accuracy across several species.
+Another research area where deep learning excels is generative modeling, where new samples are created based on the training data [@doi:10.1109/ISACV.2018.8354080]. Another area of machine learning that has been revolutionized by deep learning is reinforcement learning, which is concerned with training agents to interact with an environment [@arXiv:1709.06560].
```
Comment (Benjamin-Lee) [inline content/03.when-to-use-dl.md:37]:
```suggestion
Another research area where deep learning excels is generative modeling, where new samples are created based on the training data [@doi:10.1109/ISACV.2018.8354080].
One other area of machine learning that has been revolutionized by deep learning is reinforcement learning, which is concerned with training agents to interact with an environment [@arXiv:1709.06560].
```

**Review by Benjamin-Lee — COMMENTED** [review]
Just a few minor line breaks and one change to prevent a repetitive word

**Comment by rasbt** [comment]
> Just a few minor line breaks and one change to prevent a repetitive word

thanks!

**Pull request merged**